conformance/runner.js

```diff
@@ -105,14 +120,87 @@ function convertCommit(commitRequest) {
   return deepCopy;
 }
 
-/** Converts a Protobuf Precondition to the expected user input. */
+/** Converts a Position in Proto3 JSON to Protobuf JS format. */
+function convertPosition(position) {
+  const values = [];
+  for (let value of position.values) {
+    values.push(convert.valueFromJson(value));
+  }
+  position.values = values;
```

Comment:
Here you're mutating the provided position in-place, but below you're making a copy first. Seems like you should always make a copy first. 

FWIW- You could also ditch the for loop by using Array.map() I think.

Reply:
We don't access this state afterwards, so it was fine in this case. To make the life for future Sebastian easier, I followed your advise though and made a copy.